packages/app/src/LogSidePanelElements.tsx

```diff
@@ -0,0 +1,333 @@
+import * as React from 'react';
```

Comment:
I'm curious how do we differentiate what should belong here vs LogSidePanel? Is this like a generic util elements for the LogSidePanel?

Reply:
yep, that's the idea